services/commands_automator_service.py

```
@@ -1,159 +1,178 @@
-        if 'other_script_name_as_input' in self.scripts_attributes[script_name]:
-            return self.scripts_attributes[script_name]['other_script_name_as_input']
-        else:
-            return None
-
-
-    @staticmethod
-    def is_dir_empty(dir_path):
-        return not next(os.scandir(dir_path), None)
-
-   
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)